fabtests/pytest/efa/efa_common.py

```diff
@@ -51,3 +49,19 @@ def has_gdrcopy(hostname):
     command = "ssh {} /usr/sbin/lsmod | grep gdrdrv".format(hostname)
     process = subprocess.run(command, shell=True, check=False, stdout=subprocess.PIPE)
     return process.returncode == 0
+
+def efa_retrieve_gid(hostname):
```

Comment:
How does this work with multiple efa devices?

Reply:
This function always return the gid of the 1st EFA device.